src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -293,129 +294,141 @@ export class BrowsingContextImpl {
 
   onTargetInfoChanged(params: Protocol.Target.TargetInfoChangedEvent) {
     this.#url = params.targetInfo.url;
+
+    if (this.#navigationStarted) {
+      this.#eventManager.registerEvent(
+        {
+          method: BrowsingContext.EventNames.NavigationStarted,
+          params: {
+            context: this.id,
+            navigation: null,
+            timestamp: BrowsingContextImpl.getTimestamp(),
+            url: this.#url,
+          },
+        },
+        this.id
+      );
+      this.#navigationStarted = false;
```

Comment:
I guess there are other cases when the navigation stopped, like [`Page.frameStoppedLoading`](https://chromedevtools.github.io/devtools-protocol/tot/Page/#event-frameStoppedLoading).

Reply:
please try to make sure there are no other events pointing the navigation stopped

Reply:
It looks correct that the [Page.frameStoppedLoading](https://chromedevtools.github.io/devtools-protocol/tot/Page/#event-frameStoppedLoading) is the only event that signals that the navigation has stopped.